handlers/main.yml

```diff
@@ -1,3 +1,4 @@
 # SPDX-License-Identifier: MIT
 ---
-# Put handlers for tasks here.
+- name: Load softdog module
+  shell: modprobe softdog
```

Comment:
There is a `modprobe` module - https://docs.ansible.com/ansible/latest/collections/community/general/modprobe_module.html#ansible-collections-community-general-modprobe-module
This will unfortunately introduce a dependency on community.general in meta/requirements.yml

or - I suggest using `command` instead of `shell`

Is `softdog` present on all EL7+ and Fedora platforms?

Reply:
Thanks for pointing out - I would also suggest not using the module as it's just a simple task and adding another dependency for this doesn't look senseful to me. Will use `command` instead.
`softdog` is available on Fedora and EL7 and newer - it's part of the kernel.